composer.json

```diff
@@ -2,17 +2,17 @@
     "name": "jamescowie/composer-patcher",
     "description": "Apply patches using composer",
     "license": "MIT",
-    "version": "1.0.9",
+    "version": "1.10.0",
```

Comment:
Can we change the version to 1.0.10? Changing from 1.0.9 to 1.10.0 has a big difference. The next higher version should be either 1.0.10 or 1.1.0.

Reply:
My mistake... this should be fixed now